### DISCUSSION TITLE
How do I activate pass through?

### DISCUSSION THREAD

**Discussion by ratozumbi** [icosa-gallery/open-brush#389] [Q&A] [answered]
I undestand there is a way to draw while I see the real world. I gess it should br under environment, but I have messed with those settings and have found nothing. No lucky on google either.

Do you know? Please I need directions.

**Comment by mikeskydev** [comment, marked as answer]
You are correct, it's under environment. The first button in the list of environments looks like a headset, and will be titled 'Passthrough' when you hover over it. If you can't see that button, you're not on the latest version!
